07-sampling.Rmd

```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
+  labs(x = "Sample proportion", 
+       title = "Histogram of 33 sample proportions") 
 ```
-```{r samplingdistribution-tactile, echo=FALSE, fig.cap="Distribution of 33 proportions based on 33 samples of size 50.", fig.height=3.1, purl=FALSE}
+```{r samplingdistribution-tactile, echo=FALSE, fig.cap="The distribution of sample proportions based on 33 random samples of size 50.", fig.height=3.1, purl=FALSE}
 tactile_histogram <- ggplot(tactile_prop_red, aes(x = prop_red)) +
   geom_histogram(binwidth = 0.05, boundary = 0.4, color = "white")
 tactile_histogram +
   labs(
-    x = "Proportion of 50 balls that were red",
-    title = "Distribution of 33 proportions red"
+    x = "Sample proportion",
+    title = "Histogram of 33 sample proportions"
   )
 ```
 
 
-### What did we just do? {#sampling-what-did-we-just-do}
-
-What we just demonstrated in this activity is the statistical concept of \index{sampling} *sampling*. We would like to know the proportion of the bowl's balls that are red. Because the bowl has a large number of balls, performing an exhaustive count of the red and white balls would be time-consuming. We thus extracted a *sample* of 50 balls using the shovel to make an *estimate*. Using this sample of 50 balls, we estimated the proportion of the *bowl's* balls that are red to be 34%.
+### Recap, some terminology, and additional remarks {#sampling-what-did-we-just-do}
 
-Moreover, because we mixed the balls before each use of the shovel, the samples were randomly drawn. Because each sample was drawn at random, the samples were different from each other. Because the samples were different from each other, we obtained the different proportions red observed in Figure \@ref(fig:samplingdistribution-tactile). This is known as the concept of *sampling variation*. \index{sampling!variation}
+The activity introduced the concept of \index{sampling} *sampling*. We wanted to know the proportion of red balls in a bowl. Since the bowl had many balls, performing an exhaustive count of red and white balls was impractical. Instead, we first took a *sample* of 50 balls and determine that the *sample proportion* was 34%. This is an *estimate* of the proportion of red balls in the entire bowl. We then took more samples and obtain more sample proportions, 33 in total. This activity is what in statistics we call *sampling*. 
 
-The purpose of this sampling activity was to develop an understanding of two key concepts relating to sampling: 
+Since we returned the observed balls every time before taking a new sample we were *sampling with replacement*, and since we mixed the balls before taking each sample and then used a shovel, the samples were *randomly drawn*. When this happens we call them *random samples*. As shown in Figure \@ref(fig:samplingdistribution-tactile), different random samples produce different sample proportions. This phenomenon is called *sampling variation* and it is a central concept in this chapter. \index{sampling!variation}
```

Comment:
```suggestion
Since we returned the observed balls every time before taking a new sample we were *sampling with replacement*. Since we mixed the balls before taking each sample and then used a shovel, the samples were *randomly drawn*. When this happens we call them *random samples*. As shown in Figure \@ref(fig:samplingdistribution-tactile), different random samples produce different sample proportions. This phenomenon is called *sampling variation* and it is a central concept in this chapter. \index{sampling!variation}
```